src/app/features/template-export/template-export-selection-dialog/template-export-selection-dialog.component.html

```
@@ -1,56 +1,143 @@
+              Generated {{ progress().total }} of {{ progress().total }} files.
+            </p>
+          } @else {
+            <p i18n>
+              Generated {{ progress().total - failures().length }} of
+              {{ progress().total }} files.
+            </p>
+            <p i18n>
+              Failed:
+              {{
+                failures()
+                  .map((f) => f.entity.toString())
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)